api/cloud_api/util/SceGoogleApiHandler.js

```diff
@@ -41,16 +42,19 @@ class SceGoogleApiHandler {
     );
     this.oAuth2Client.setCredentials({
       // eslint-disable-next-line
-      refresh_token: REFRESH_TOKEN
+        refresh_token: REFRESH_TOKEN
     });
+    if(CLIENT_ID != 'NOT_SET' && CLIENT_SECRET != 'NOT_SET') {
+      this.hasValidApiKeys = true;
+    }
   }
 
   /**
-   * Checks if the token file from the given path from the constructor exists.
-   * It will either resolve a boolean false or an object containing the token
-   * data.
-   * @returns {Promise<boolean|Object>} The result of the token file search.
-   */
+     * Checks if the token file from the given path from the constructor exists.
```

Comment:
this file is full of random changes, i would turn ur autoformatter off and undo these